controller/controller.go

```diff
@@ -0,0 +1,107 @@
+package controller
+
+import (
+  "fmt"
+  "net/http"
+  "encoding/json"
+  "github.com/gorilla/mux"
+  "strconv"
+  "github.com/sirupsen/logrus"
+  "github.com/unrolled/render"
+  "github.com/halarcon-wizeline/academy-go-q12021/domain"
+  "github.com/halarcon-wizeline/academy-go-q12021/infrastructure/datastore"
+  "github.com/halarcon-wizeline/academy-go-q12021/infrastructure/externalapi"
+)
+
+// UseCase interface
+type UseCase interface {
```

Comment:
Are you using this interface somewhere?

Reply:
It's not used, it will be fixed in the next deliverable